enhancements for fax requests in request manager

Kocma-san opened the pull request:
## About The Pull Request
Addition to [#84910](https://github.com/tgstation/tgstation/pull/84910)

Added a "print" button to the request manager. It simply prints at the centcom station the paper that was faxed to the admins.
Also added "Auto-print Faxes" button to the request manager, which allows you to enable automatic printing of requests on the admin fax
<details>
<summary>Screenshots</summary>

![image](https://github.com/user-attachments/assets/35cbda0a-c759-4e9e-8899-8e2d81069b4c)
![image](https://github.com/user-attachments/assets/bbb918da-8b84-4a6b-a42e-c06359ab5651)
![image](https://github.com/user-attachments/assets/b380cc13-a9e2-496e-a296-60fb827c4c55)

</details>

## Why It's Good For The Game

The message in the chat may get lost. In this case, it will no longer be possible to print the paper sent to the CC
Changing the allow_exotic_faxes variable will allow you to fax something unusual to the station in (very rare) situations. No need to manually change this variable
## Changelog
:cl:
add: added a "print" button to the request manager
add: admin fax can now send exotic items
add: added "Auto-print Faxes" button to the request manager, which allows you to enable automatic printing of requests on the admin fax
fix: fixed a bug where pressing the print button would cause the receive animation to appear on all admin faxes.
/:cl:

Kocma-san committed:
weh

Kocma-san committed:
weh2

Kocma-san committed:
weh2.5

jlsnow301 commented:
buttons are overlapping with the section title. can you move filterpanel outside of the section buttons? perhaps into the content?

Kocma-san commented:
> buttons are overlapping with the text, see picture. Can you move them elsewhere?

I will think what I can do. But only if `tgui-dev` works for me (I dont understand why it stopped working for me)

Kocma-san committed:
Merge remote-tracking branch 'upstream/master' into faxes-(again)

Kocma-san committed:
conflict resolved

Kocma-san committed:
changes

- fixed a bug that when pressing the print button, the animation of receiving was for all admin faxes (almost impossible situation, but...)
- added "Auto-print Faxes" button to the request manager, which allows you to enable automatic printing of requests on the admin fax

Kocma-san committed:
Boolean -> boolean

Kocma-san committed:
toggleprit -> toggleprint

jlsnow301 commented on tgui/packages/tgui/interfaces/RequestManager.tsx:
```diff
@@ -12,6 +12,7 @@ import { Window } from '../layouts';
 
 type Data = {
   requests: Request[];
+  fax_autoprinting: boolean;
```
`BooleanLike`

Kocma-san committed:
boolean -> BooleanLike
